app/controllers/custom_labels_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class CustomLabelsController < ApplicationController
+  def index
+    render jsonapi: custom_labels.all, include: params[:include]
+  end
+
+  def update
+    data_attributes = JSON.parse(request.body.read)['data']['attributes'] || {}
```

Comment:
you're gonna want to throw a `rescue` at the bottom of the update method to catch any `JSON:ParserError` that may be thrown due to a malformed request body.  there's an example of this in `configurations_controller.rb` that should be pretty easy to port over

Reply:
sure, yes, good point, thanks!